server/channels/app/platform/enterprise.go

```diff
@@ -37,3 +37,9 @@ var metricsInterfaceFn func(*PlatformService, string, string) einterfaces.Metric
 func RegisterMetricsInterface(f func(*PlatformService, string, string) einterfaces.MetricsInterface) {
 	metricsInterfaceFn = f
 }
+
+var pdpInterface func(*PlatformService) einterfaces.PolicyDecisionPointInterface
+
+func RegisterPdpInterface(f func(*PlatformService) einterfaces.PolicyDecisionPointInterface) {
+	pdpInterface = f
```

Comment:
question: should this function maybe have a safeguard that checks if the interface has been already registered?

Reply:
We can do that but this function will be called with `init()` so it's guaranteed that would be called only once by runtime so it wouldn't be necessary.